Shared/ModuleHandle.ps1

```diff
@@ -29,19 +29,18 @@ function Request-Module {
     [OutputType([bool])]
     param (
         [Parameter(Mandatory = $true)]
-        [string[]]$Modules,
+        [string[]]$Module,
         [Parameter(Mandatory = $false)]
         [System.Version]$MinModuleVersion = $null
     )
 
     $noFoundError = $true
     foreach ($module in $Modules) {
```

Comment:
You broke this loop because you changed the parameter name to be `$Module` instead of `$Modules`.

Reply:
Thanks for point to this, I am modifying the parameter to singular as you recommend in a previous PR from the Development Guidelines.
I will fix this and rebase